src/api/response_models/request.py

```diff
@@ -29,5 +30,9 @@ def parse_from(cls, request_obj: Request) -> RequestResponse:
             date_of_birth=request_obj.user.date_of_birth,
             city=request_obj.user.city,
             phone_number=request_obj.user.phone_number,
-            status=request_obj.status,
+            request_status=request_obj.status,

```

Comment:
Я бы изменил название объекта на obj. Аннотация типов говорит какой тип объекта. А так больше визуального мусора, который не добавляет понятности коду. 
```suggestion
            request_id=obj.id,
            user_id=obj.user.id,
            name=obj.user.name,
            surname=obj.user.surname,
            date_of_birth=obj.user.date_of_birth,
            city=obj.user.city,
            phone_number=obj.user.phone_number,
            request_status=obj.status,
```